testkit-backend/backend.go

```diff
@@ -392,6 +392,13 @@ func (b *backend) handleRequest(req map[string]interface{}) {
 			if data["resolverRegistered"].(bool) {
 				c.AddressResolver = b.customAddressResolverFunction()
 			}
+			if data["connectionAcquisitionTimeoutMs"] != nil {
+				timeout := time.Millisecond * time.Duration(data["connectionAcquisitionTimeoutMs"].(float64))
```

Comment:
nit: you can probably inline the variable for consistency's sake

Reply:
I could. The line just got ugly long. Do you prefer it in one line?

Reply:
Go for it! 
![long line](https://static.independent.co.uk/2020/09/22/09/Untitled.png?width=1200)

Reply:
Done

![it's long now!](https://www.catster.com/wp-content/uploads/2015/06/600px-iwo-longcat.jpg)